api/src/main/java/jakarta/data/page/Slice.java

```diff
@@ -21,16 +21,36 @@
 import java.util.List;
 
 /**
- * A slice of data that indicates whether there's a next or previous slice available.
+ * <p>A slice contains the data that is retrieved for a page request.
+ * A slice is obtained by supplying a {@link Pageable} parameter to a repository method.
+ * For example,</p>
  *
- * @param <T> the type of elements in this slice 
+ * <pre>
+ * {@code Slice<Vehicle>} find({@code @By("make")} String make,
+ *                     {@code @By("model")} String model,
+ *                     {@code @By("year")} int designYear,
+ *                     Pageable pagination);
+ * </pre>
+ *
+ * <p>Unlike {@link Page}, a {@code Slice} does not have awareness of the total number of pages
+ * that can be retrieved for the query.</p>
+ *
+ * <p>The lowercase terms, <i>slice</i> and <i>page</i>, are used interchangeably
+ * throughout this specification. Wherever a distinction needs to be made between them,
+ * the class name, {@code Slice} or {@code Page}, is used.</p>
+ *
+ * @param <T> the type of elements in this slice.
  */
 public interface Slice<T> extends Streamable<T> {
 
     /**
-     * Returns the page content as {@link List}.
+     * Returns the page content as a {@link List}.
+     * The list is sorted according to the combined sort criteria of the repository method
+     * and the sort criteria of the page request that is supplied to the repository method,
+     * sorting first by the sort criteria of the repository method,
+     * and then by the sort criteria of the page request.
      *
-     * @return the page content as {@link List}; will never be {@literal null}.
+     * @return the page content as a {@link List}; will never be {@literal null}.
```

Comment:
```suggestion
     * @return the page content as a {@link List}; will never be {@code null}.
```
Should this be `@code` instead of `@literal`? null doesn't seem to need `@literal`, at least not how VSCode renders the javadoc, but `@code` could make it clear what we mean by not returning null.